python/GafferUI/UIEditor.py

```diff
@@ -264,6 +276,7 @@ def __updateFromSetInternal( self, lazy=True ) :
 				# is available for use by the user on Reference nodes once a Box has
 				# been exported and referenced.
 				plugParent = self.__node
+				self.__plugAddButtons.setVisible( True )
```

Comment:
@johnhaddon Can you see any issues in doing this for all `Box` derived classes rather than `type(node) == Gaffer.Box`?